scripts/01858-Sello.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+	Author: Sello C. Fotoyi
+	File: 01858-Sello.php
+	Description: script which outputs a "Hello world",
+				 along with author's name and HNGi7 ID,
+				  as requested for HNGi7 Task 2.
+	Date: 03 June 2020
+
+*/
+
+	define("FULL_NAME", "Sello Clinton Fotoyi",true);
+	define("HNG_ID","HNG-01858",true);
+	$language = "php";
+	
+	echo "Hello World, this is ".FULL_NAME." with HNGi7 ID ".HNG_ID;
+	echo " using ".$language " for stage 2 task.";
```

Comment:
Please append a space and your email behind the word `task` as described [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)